docs/file-formats/xml-data-file.md

```diff
@@ -198,6 +198,7 @@ Example of a property element of type integer with two values:
 The following property elements exist:
 
 - `<bitstream>`: contains a path to a file (if the resource is a multimedia resource)
+- `<iiif-uri>`: contains a URI to an IIIF image server (resource must be a `StillImageRepresentation`)
```

Comment:
the part in brackets should be identical to the line above. You can change either the line above, or this line, but I find it confusing if the same thing is described in 2 different ways

Reply:
Sorry, my fault. Sure, this is correct, you can leave it as it is 👍